uma/protocol/payreq.py

```diff
@@ -71,6 +71,11 @@ class PayRequest(JSONable):
     The major version of the UMA protocol that this currency adheres to. This is not serialized to JSON.
     """
 
+    invoice_uuid: Optional[str] = None
+    """
+    The invoice UUID that the sender is paying.
```

Comment:
General note: should we be explicit that this is the UMA Invoice, and not like the bolt11 invoice, etc?

Reply:
I think for the field name I'll keep it as invoice_uuid but we should add to the comment to say it's uma invoice.